runtime/Go/antlr/token.go

```diff
@@ -14,6 +14,10 @@ type TokenSourceCharStreamPair struct {
 	charStream  CharStream
 }
 
+func NewTokenSourceCharStreamPair(source TokenSource, stream CharStream) *TokenSourceCharStreamPair {
```

Comment:
As far and I remember this is an old change in an already accepted PR. I'll check, a rebase might give ride of it. If memory servers this was need to enable a custom lexer (eg a block based lexer)